src/components/EventCalender/Header/EventHeader.tsx

```
@@ -34,11 +34,12 @@
  */
 
 import React from 'react';
+import Button from 'shared-components/Button';
```

ESLint

[error] 'shared-components/Button' imported multiple times.

(import/no-duplicates)